src/libs/CMakeLists.txt

```diff
@@ -2,6 +2,17 @@
 # Project developers. See top-level LICENSE AND COPYRIGHT files for dates and
 # other details. No copyright assignment is required to contribute to Conduit.
 
+# Create an interface target to hold the common flags for python targets
+add_library(conduit_python_build INTERFACE)
+install(TARGETS conduit_python_build EXPORT "conduit")
+
+#set(CONDUIT_PYTHON_LIMITED_API)
+set(CONDUIT_PYTHON_LIMITED_API 0x030A0000)
+
+if(CONDUIT_PYTHON_LIMITED_API)
```

Comment:
This variable should be guarded by a version check of Python itself that it is of at least 3.10 as well (or whatever the limited API version ends up being).

Reply:
good point !